### PR TITLE
Fix/tab panel

### PR DESCRIPTION
**Pull request by San88Francisco**

### CODE REVIEW

File: src/components/Account-panel/style.ts [hunk 1]
```diff
@@ -4,3 +4,15 @@ export const stInput = {
...
+  fontFamily: 'Comfortaa',
+  fontSize: '32px',
+  fontWeight: 500
+}
+
+export const stP = {
+  fontFamily: 'Open Sans',
+  fontSize: '18px',
+  fontWeight: 300
+}
```
Comment (pqoqubbw) [inline src/components/Account-panel/style.ts]: you don't need this styles, we already have this styles in typography variants

File: src/components/Layout/breadcrumbs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ const Breadcrumbs = () => {
     (state: RootState) => state.items.allCategories
   )
   const allProducts = useSelector((state: RootState) => state.items.items)
+  console.log('✌️allProducts --->', allProducts)
```
Comment (pqoqubbw) [inline src/components/Layout/breadcrumbs/Breadcrumbs.tsx]: remove console.log

File: src/components/Modal-custom-btn/ModalCustomBtnEmail.tsx [hunk 3]
```diff
@@ -3,18 +3,11 @@ import { Link } from 'react-router-dom'
...
-  stIconEmail,
-  stIconLinkEmail,
-  stLinkEmail,
-  stLinkEmailList,
-  stLinkEmailP
-} from './style'
+
 import { useSnackbar } from '../../hooks/useSnackbar'
+import { btn, emailP2, iconLinkEmail, linkEmailList } from './style'
+import Modalka from './ModalCustomWindow'
```
Comment (pqoqubbw) [inline src/components/Modal-custom-btn/ModalCustomBtnEmail.tsx]: that is wrong name for component, use english words

File: src/components/Modal-custom-btn/ModalCustomWindow.tsx [hunk 4]
```diff
@@ -2,15 +2,15 @@ import React from 'react'
...
+import { boxModWin } from './style'
 type TModalWindow = {
   yourStBoxModalWindow?: React.CSSProperties
   children: React.ReactNode
   openModal: boolean
   setOpenModal: React.Dispatch<React.SetStateAction<boolean>>
 }
 
-export default function InfoConfirmationModal({
+export default function Modalka({
```
Comment (pqoqubbw) [inline src/components/Modal-custom-btn/ModalCustomWindow.tsx]: same here, also don't use `export default`, use `export { ComponentName }`

File: src/pages/account-page/account-page.tsx [hunk 5]
```diff
@@ -44,68 +36,57 @@ function a11yProps(index: number) {
...
+              disableRipple
+              label={`Контактна інформація`}
+              {...a11yProps(0)}
+            />
+            <Tab
+              disableRipple
+              label={`Ваші адреси доставки`}
+              {...a11yProps(1)}
+            />
+            <Tab disableRipple label={`Історія замовлень`} {...a11yProps(2)} />
```
Comment (pqoqubbw) [inline src/pages/account-page/account-page.tsx]:
```suggestion
            <Tab disableRipple label="Історія замовлень" {...a11yProps(2)} />
```

File: src/pages/account-page/account-page.tsx [hunk 6]
```diff
@@ -44,68 +36,57 @@ function a11yProps(index: number) {
...
+            <Tab disableRipple label={`Змінити пароль`} {...a11yProps(3)} />
```
Comment (pqoqubbw) [inline src/pages/account-page/account-page.tsx]:
```suggestion
            <Tab disableRipple label="Змінити пароль" {...a11yProps(3)} />
```

File: src/pages/account-page/account-page.tsx [hunk 7]
```diff
@@ -44,68 +36,57 @@ function a11yProps(index: number) {
...
+        <Box sx={{ p: '40px 0' }}>
+          <Tabs
+            value={value}
+            onChange={(_, newValue) => setValue(newValue)}
+            aria-label="basic tabs example"
+            sx={stTabsNav}
+          >
+            <Tab
+              disableRipple
+              label={`Контактна інформація`}
```
Comment (pqoqubbw) [inline src/pages/account-page/account-page.tsx]:
```suggestion
              label="Контактна інформація"
```

File: src/pages/account-page/account-page.tsx [hunk 8]
```diff
@@ -44,68 +36,57 @@ function a11yProps(index: number) {
...
+              {...a11yProps(0)}
+            />
+            <Tab
+              disableRipple
+              label={`Ваші адреси доставки`}
```
Comment (pqoqubbw) [inline src/pages/account-page/account-page.tsx]:
```suggestion
              label="Ваші адреси доставки"
```

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/Account-panel/style.ts (hunk 1 above)
Comment (San88Francisco) [reply src/components/Account-panel/style.ts]: done

File: src/components/Layout/breadcrumbs/Breadcrumbs.tsx (hunk 2 above)
Comment (San88Francisco) [reply src/components/Layout/breadcrumbs/Breadcrumbs.tsx]: done

File: src/components/Modal-custom-btn/ModalCustomBtnEmail.tsx (hunk 3 above)
Comment (San88Francisco) [reply src/components/Modal-custom-btn/ModalCustomBtnEmail.tsx]: done

File: src/components/Modal-custom-btn/ModalCustomWindow.tsx (hunk 4 above)
Comment (San88Francisco) [reply src/components/Modal-custom-btn/ModalCustomWindow.tsx]: done

File: src/pages/account-page/account-page.tsx (hunk 5 above)
Comment (San88Francisco) [reply src/pages/account-page/account-page.tsx]: done

File: src/pages/account-page/account-page.tsx (hunk 6 above)
Comment (San88Francisco) [reply src/pages/account-page/account-page.tsx]: done

File: src/pages/account-page/account-page.tsx (hunk 7 above)
Comment (San88Francisco) [reply src/pages/account-page/account-page.tsx]: done

File: src/pages/account-page/account-page.tsx (hunk 8 above)
Comment (San88Francisco) [reply src/pages/account-page/account-page.tsx]: done

File: src/components/Modal-custom-btn/ModalCustomBtnEdit.tsx [hunk 9]
```diff
@@ -1,84 +1,88 @@
...
+            variant="outlined"
+            onClick={handleClickLogout}
+            sx={{
+              ...btnEditModWin,
+              ...{
+                width: 250,
+                height: 50,
+                textTransform: 'capitalize'
+              }
+            }}
```
Comment (pqoqubbw) [inline src/components/Modal-custom-btn/ModalCustomBtnEdit.tsx]:
```suggestion
            sx={{
              ...btnEditModWin,
                width: 250,
                height: 50,
                textTransform: 'capitalize'
            }}
```

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/Modal-custom-btn/ModalCustomBtnEdit.tsx (hunk 9 above)
Comment (San88Francisco) [reply src/components/Modal-custom-btn/ModalCustomBtnEdit.tsx]: done

**Review by pqoqubbw — APPROVED** [review]

**Pull request merged**